x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/dataframe/analyses/OutlierDetectionTests.java

```diff
@@ -41,20 +48,38 @@ public static OutlierDetection createRandom() {
     }
 
     public void testGetParams_GivenDefaults() {
-        OutlierDetection outlierDetection = new OutlierDetection();
-        assertThat(outlierDetection.getParams().isEmpty(), is(true));
+        OutlierDetection outlierDetection = new OutlierDetection.Builder().build();
+        Map<String, Object> params = outlierDetection.getParams();
+        assertThat(params.size(), equalTo(3));
+        assertThat(params.containsKey("compute_feature_influence"), is(true));
```

Comment:
FYI: There are `hasKey` and `hasEntry` methods in `org.hamcrest.Matchers` which you could use here.